.github/workflows/wheels.yaml

```diff
@@ -164,11 +164,6 @@ jobs:
         os: [ubuntu-latest, macos-13, macos-14]
 
     steps:
-      - uses: actions/download-artifact@v4
-        with:
-          name: artifact
-          path: dist
-
```

Comment:
Ok, can see why this isn't necessary, at least if we want to check that test-pypi is working.